tests/displayBooksListTest.php

```diff
@@ -26,15 +26,16 @@ public function test_success_display_books_list()
         $book = [new Book(
             1,
             'Title',
+            1,
```

Comment:
I'm getting a test failure due to this, it looks like the second param of Book should be the id, but here you're passing in the title. Looks like the params are mismatched slightly

Reply:
TypeError: Collection\Entities\Book::__construct(): Argument #2 ($author_id) must be of type int, string given